tests/e2e/provisioning/Dockerfile

```
@@ -0,0 +1,19 @@
+FROM golang:1.12.5-alpine3.9 as builder
+
+ENV SRC_DIR=/go/src/github.com/kyma-incubator/compass/tests/e2e/provisioning
+
+WORKDIR $SRC_DIR
+COPY . $SRC_DIR
+
+RUN CGO_ENABLED=0 GOOS=linux go test -c ./test
```

```suggestion
RUN CGO_ENABLED=0 GOOS=linux go test -ldflags="-s -w" -c ./test
```

with `-ldflags="-s -w"` you will reduce ~15mb on that binary